polytiles/polytiles.py

```diff
@@ -23,30 +23,28 @@
 except ImportError:
     HAS_SQLITE = False
 
-DEG_TO_RAD = pi/180
-RAD_TO_DEG = 180/pi
+DEG_TO_RAD = pi / 180
+RAD_TO_DEG = 180 / pi
 TILE_SIZE = 256
 LIST_QUEUE_LENGTH = 32
 
 
 def box(x1, y1, x2, y2):
     return Polygon([(x1, y1), (x2, y1), (x2, y2), (x1, y2)])
 
-
 def minmax(a, b, c):
     a = max(a, b)
     a = min(a, c)
     return a
 
-
 def format2ext(format):
-    ext = 'png'
-    if format.startswith('jpeg'):
-        ext = 'jpg'
-    elif format.startswith('tif'):
-        ext = 'tif'
-    elif format.startswith('webp'):
-        ext = 'webp'
+    ext = "png"
```

Comment:
Was there any reason to change quotes to double? They are the same in Python, so the only guideline is to have consistent quotes.